manifests/profile/server.pp

```diff
@@ -390,9 +395,9 @@
     }
 
     St2::Package::Install<| tag == 'st2::profile::server' |>
-    -> Ini_setting<| tag == 'st2::profile::server' |>
+    -> Ini_setting<| tag == 'st2::config' |>
     -> Exec['start st2']
```

Comment:
This command name got me confused for a second (start instead of restart), but it actually restarts the services:

```
    exec { 'start st2':
      command => 'st2ctl restart',
      unless  => 'ps ax | grep -v grep | grep actionrunner',
      path    => '/usr/bin:/usr/sbin:/bin:/sbin',
      require => Exec['register st2 content'],
    }
```

Anyway, not a blocker for this PR.